tests/common/mac/ProjectTestHelpers.cs

```diff
@@ -147,6 +147,25 @@ public static string BuildProject (string csprojTarget, bool isUnified, bool dia
 				return RunAndAssert ("/Applications/Visual Studio.app/Contents/MacOS/vstool", buildArgs, "Compile", shouldFail, getBuildProjectErrorInfo);
 		}
 
+		public static string Quote (string f)
```

Comment:
We already have a lot of Quote implementations, maybe we should stop copying this everywhere and create a single file for it?

```shell
$ git grep public.*string.*Quote
msbuild/Xamarin.MacDev.Tasks.Core/ProcessArgumentBuilder.cs:            public static string Quote (string text)
src/generator.cs:       public static string Quote (string s)
tests/mtouch/MTouch.cs:         public static string Quote (string f)
tests/xharness/Harness.cs:              public static string Quote (string f)
tools/mmp/aot.cs:               public string Quote (string f) => Driver.Quote (f);
tools/mmp/driver.cs:            public static string Quote (string f)
tools/mtouch/mtouch.cs:         public static string Quote (string f)
```

And these are not all identical:

```
tests/mtouch/MTouch.cs-     if (f.IndexOf (' ') == -1 && f.IndexOf ('\'') == -1 && f.IndexOf (',') == -1 && f.IndexOf ('\\') == -1)
tests/xharness/Harness.cs-  if (f.IndexOf (' ') == -1 && f.IndexOf ('\'') == -1 && f.IndexOf (',') == -1)
```

and none of these match your implementation either.